plugins/Fable.React.Plugins.fs

```diff
@@ -79,6 +79,9 @@ module internal AstUtil =
         Fable.ObjectExpr(List.map objValue kvs, Fable.Any, None)
 
 
+[<assembly:ScanForPlugins>]
+do()
```

Comment:
I found a way to avoid having to scan all assemblies, so this is necessary now to tell Fable that this assembly in particular does contain plugins and should be scanned. Also, please put the plugins directly in a namespace and not inside a module.